### ISSUE TITLE
Fail to build on macOS M1 

### ISSUE THREAD

**Issue by djmango** [zowens/crc32c#62]
The latest nightly fails to build with `error: could not compile crc32c (lib) due to 1 previous error` `unknown feature stdarch_arm_crc32`

took a look at this:
https://github.com/zowens/crc32c/issues/58

downgrading to `nightly-2024-02-08` works but is not optimal

**Comment by sassman** [comment]
I guess it is related to the stabilization for stdarch_arm_crc32

see: https://github.com/rust-lang/stdarch/pull/1573#discussion_r1595987982

There they suggest to feature gate it that way:

```
#[cfg_attr(target_arch = "arm", unstable(feature = "stdarch_aarch32_crc32", issue = "XXXX"))]
#[cfg_attr(not(target_arch = "arm"), stable(feature = "stdarch_aarch64_crc32", since = "1.80.0"))]
```

**Comment by zowens** [comment]
Should be fixed with the 0.6.6 release. Can you confirm and close?

**Comment by djmango** [comment]
looks good, thanks all!
